easybuild/easyconfigs/f/FLUENT/FLUENT-2021R2.eb

```diff
@@ -0,0 +1,17 @@
+name = 'FLUENT'
+version = '2021R2'
+
+homepage = 'http://www.ansys.com/Products/Simulation+Technology/Fluid+Dynamics/Fluid+Dynamics+Products/ANSYS+Fluent'
```

Comment:
```suggestion
homepage = 'https://www.ansys.com/Products/Simulation+Technology/Fluid+Dynamics/Fluid+Dynamics+Products/ANSYS+Fluent'
```

Reply:
https://www.ansys.com/products/fluids/ansys-fluent  seems to be more up to date